scripts/apps/search/MultiImageEdit.ts

```diff
@@ -196,6 +196,8 @@ export function MultiImageEditDirective(asset, $sce) {
         templateUrl: asset.templateUrl('apps/search/views/multi-image-edit.html'),
         link: function(scope) {
             scope.trustAsHtml = $sce.trustAsHtml;
+            scope.metadataFromStorage = localStorage.getItem('metadata:items');
```

Comment:
would be good to define constants for those storage keys

Reply:
Where should I define them? Should I put it on top like ```CONST metadataItems = "metadata:item"```?

Reply:
top of file or just top of link, doesn't matter. and should  be more like:
```
const METADATA_ITEMS = 'metadata:items';
const METADATA_AUTOFILL = 'metadata:autofill';
```

Reply:
done 😄